src/vm/amd64/cgencpu.h

```diff
@@ -234,6 +238,19 @@ struct ArgumentRegisters {
     #undef ARGUMENT_REGISTER
 };
 
+typedef DPTR(struct CalleeSavedRegisters) PTR_CalleeSavedRegisters;
+struct CalleeSavedRegisters {
+    #define CALLEE_SAVED_REGISTER(regname) INT_PTR regname;
+    ENUM_CALLEE_SAVED_REGISTERS();
+    #undef CALLEE_SAVED_REGISTER
+};
+
+struct CalleeSavedRegistersPointers {
+    #define CALLEE_SAVED_REGISTER(regname) PTR_TADDR regname;
```

Comment:
I think this needs to be p##regname instead of regname like in other places. Otherwise it will cause a build break.